models/tags.go

```diff
@@ -45,12 +45,31 @@ type CreateTagsSchema struct {
 	EntityName string      `json:"entity_name"`
 }
 
+type CreateNewTagSchema struct {
+	Name       string `json:"name" binding:"required,min=1,max=20"`
+	Color      string `json:"color"`
+	EntityType string `json:"entity_type"`
+	EntityName string `json:"entity_name"`
+}
+
 type RemoveTagsSchema struct {
 	Names      []string `json:"names"`
 	EntityType string   `json:"entity_type"`
 	EntityName string   `json:"entity_name"`
 }
 
+type EditTagsSchema struct {
+	TagsToAdd    []Tag              `json:"tags_to_add"`
```

Comment:
this is edit so why tags to add

Reply:
it's edit tags of entity - you get an array of tags to add and array of tags to remove - I'll change it to EditTagsListSchema